static/set_proxy.sh

```diff
@@ -128,88 +122,41 @@ fi
 echo "Git proxy set"
 }
 
-#setting docker proxy
```

Comment:
Why have you removed docker proxy?

Reply:
While merging with the upstream repository, I didn't address the conflicts, so my repository kept my version of the script(without docker function).